gradle.properties

```diff
@@ -11,7 +11,6 @@
 # The setting is particularly useful for tweaking memory settings.
 android.useAndroidX=true
 org.gradle.jvmargs=-Xms1g -Xmx4g
-android.enableR8=true
```

Comment:
Was this intentional? (just curious!)

Reply:
Yes, in order to use the glean plugin, we needed Focus to upgrade to a newer gradle plugin, that plugin required us removing deprecated properties.

This isn't an issue with Glean, but rather how outdated the Focus build tools are since we last updated them. 🙂

Reply:
I've moved that commit into a [separate PR](https://github.com/mozilla-mobile/focus-android/pull/4905) so that it could land sooner.